pyfixest/estimation/estimation.py

```diff
@@ -13,15 +13,15 @@
     WeightsTypeOptions,
 )
 from pyfixest.utils.dev_utils import DataFrameType
-from pyfixest.utils.utils import ssc
+from pyfixest.utils.utils import ssc as ssc_func
```

Comment:
This was a particularly tricky bug as `SCC` was both a global function and a function parameter.